jest-common/src/test/java/io/searchbox/indices/script/CreateIndexedScriptTest.java

```diff
@@ -31,13 +30,6 @@ public void setUp() {
                 "def test_b=\"$test_a\"\n";
     }
 
```

Comment:
This test should modified to check default script is painless

Reply:
Also we need to add a test that checks given script language value is properly set.